tigerpath/majors_and_certificates/degrees/BSE.json

```
@@ -250,18 +250,33 @@
           "min_needed": 4,
           "explanation": "The humanities and social science courses must include one course in four of the six areas listed below:\n\n- Epistemology and Cognition (EC)\n- Ethical Thought and Moral Values (EM)\n- Foreign Language (at the 107/108 level or above)\n- Historical Analysis (HA)\n- Literature and the Arts (LA)\n- Social Analysis (SA)",
           "req_list": [
+            {
+              "name": "Culture and Difference",
+              "max_counted": 1,
+              "min_needed": 1,
```

From the website you linked, it states that "For B.S.E. students this will represent an additional area in which students may elect to satisfy Humanities and Social Science distribution requirements." Here, however, the req appears to be mandatory. I believe that the `min_needed` field should be set to `null`.

---

Oh yeah, good catch! I must have copied it over from the A.B. version and forgot to change it.
```suggestion
              "min_needed": null,
```